This fixes multiple definitions of the bitbang_swd symbol and thereby
prevents a build failure with recent GCC versions.

diff -upr openocd-0.10.0.orig/src/jtag/drivers/bitbang.h openocd-0.10.0/src/jtag/drivers/bitbang.h
--- openocd-0.10.0.orig/src/jtag/drivers/bitbang.h	2016-12-25 15:12:55.000000000 +0100
+++ openocd-0.10.0/src/jtag/drivers/bitbang.h	2020-10-22 19:01:07.914466540 +0200
@@ -35,7 +35,7 @@ struct bitbang_interface {
 	void (*swdio_drive)(bool on);
 };
 
-const struct swd_driver bitbang_swd;
+extern const struct swd_driver bitbang_swd;
 
 extern bool swd_mode;
 
